### ISSUE TITLE
multisite?

### ISSUE THREAD

**Issue by tgww** [wodby/docker4drupal#69]
I do not understand how to run multisite?

**Comment by PavelPrischepa** [comment]
Just copy paste mariadb service definition

```
  mariadb1:
    image: wodby/drupal-mariadb:1.0.0
    environment:
      MYSQL_RANDOM_ROOT_PASSWORD: 1
      MYSQL_DATABASE: drupal
      MYSQL_USER: drupal
      MYSQL_PASSWORD: drupal

  mariadb2:
    image: wodby/drupal-mariadb:1.0.0
    environment:
      MYSQL_RANDOM_ROOT_PASSWORD: 1
      MYSQL_DATABASE: drupal
      MYSQL_USER: drupal
      MYSQL_PASSWORD: drupal

...
```
then specify db hostname in settings.php mariadb1 for site1, mariadb2 for site2 - and so on.

**Comment by damonteo** [comment]
the database part is fine for me but how can we set the uri for multisite instances? as localhost:8000 goes with the sites/all/default site.. 

what is the proper way to load a sites/all/some_multisite_instance?

**Comment by PavelPrischepa** [comment]
@damonteo you should use [multi-site directory aliasing feature](https://api.drupal.org/api/drupal/sites%21example.sites.php/8.2.x). For example:

Domains:

- site1.loc:8000
- site2.loc:8000

Dirs in `sites/`:

- site1
- site2

Aliases `sites/sites.php`:
```
$sites = [
  '8080.site1.loc:8000' => 'site1',
  '8080.site2.loc:8000' => 'site2',
];
```

**Comment by damonteo** [comment]
thanks a lot @PavelPrischepa , got it working. an other quick question (probably noob.. just starting with docker..):

if I have two completely separate docker projects in separate folders do I still have to add custom name  to the the services? or separate ports? as the volumes mixed up somehow

so:
project1-folder
 --docker-compose
 --sites/all/themes/ theme1, theme2
project2-folder
 --docker-compose
 --sites/all/themes/ theme3, theme4

and if I run project 1 only, I have /var/www/html/sites/all/themes/ theme1, theme2, theme3, theme4

is that normal?

**Comment by PavelPrischepa** [comment]
You should use 1 docker compose file per project and separate ports for services.

> and if I run project 1 only, I have /var/www/html/sites/all/themes/ theme1, theme2, theme3, theme4

Are you sure you mounted `project1-folder` to `/var/www/html` in nginx and php containers?

**Comment by damonteo** [comment]
actually it was caused by the unison sync. with the default config everything works fine,
but when I enable d4d-unison-sync with your default config it throws an error:

`docker-sync start
          ok  Starting unison
       error  Error starting sync, exit code 126
     message
     message  Usage: unison [options]
    or unison root1 root2 [options]
    or unison profilename [options]

For a list of options, type "unison -help".
For a tutorial on basic usage, type "unison -doc tutorial".
For other documentation, type "unison -doc topics".

unison was invoked incorrectly (too many roots)
     success  Unison server started
       error  Error starting sync, exit code 126
     message
     message  Usage: unison [options]
    or unison root1 root2 [options]
    or unison profilename [options]

For a list of options, type "unison -help".
For a tutorial on basic usage, type "unison -doc tutorial".
For other documentation, type "unison -doc topics".

unison was invoked incorrectly (too many roots)
Usage: unison [options]
    or unison root1 root2 [options]
    or unison profilename [options]

For a list of options, type "unison -help".
For a tutorial on basic usage, type "unison -doc tutorial".
For other documentation, type "unison -doc topics".

unison was invoked incorrectly (too many roots)`

when I changed the strategy to unison-unox it worked one way only and that caused the mixed folders.

so is there anything else to change for unox to work other than the docker-sync and compose files?
